osquery/tables/networking/darwin/wifi_utils.h

```diff
@@ -0,0 +1,28 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <CoreWLAN/CoreWLAN.h>
+#include <iomanip>
```

Comment:
Nit, space between OS includes and C++ includes, and space between the includes and start of code.